http/util.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"log"
+	"math/rand"
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+func randValues(max int) [6]uint8 {
+	return [6]uint8{
+		uint8(rand.Intn(max)),
+		uint8(rand.Intn(max)),
+		uint8(rand.Intn(max)),
+		uint8(rand.Intn(max)),
+		uint8(rand.Intn(max)),
+		uint8(rand.Intn(max)),
+	}
+}
```

Comment:
This might as well be `NewRandomParty` (#148)

Reply:
Yeah, it's a placeholder that needs to be replaced later.